### PR TITLE
Update the Azure publishing pipeline to output result from pub

### PR DESCRIPTION
**Pull request by jsturtevant**
This updates the script for the case where the command errors out the error and the message is lost. This now prints out any output from pub

/assign @CecileRobertMichon

### CODE REVIEW

File: images/capi/packer/azure/scripts/new-sku.sh [hunk 1]
```diff
@@ -65,7 +65,10 @@ echo "Getting pub..."
 (set -x ; curl -fsSL https://github.com/devigned/pub/releases/download/v0.2.6/pub_v0.2.6_linux_amd64.tar.gz -o pub; tar -xzf pub)
 
 echo "Creating new SKU"
-(set -x ; ./pub_linux_amd64 skus put -p $PUBLISHER -o "$OFFER" -f sku.json ; echo "")
+set -x 
+./pub_linux_amd64 skus put -p $PUBLISHER -o "$OFFER" -f sku.json 
+set +x
+echo "Created sku"
```
Comment (CecileRobertMichon) [inline images/capi/packer/azure/scripts/new-sku.sh:71]: missing newline before Create Sku

File: images/capi/packer/azure/scripts/new-sku.sh [hunk 2]
```diff
@@ -65,7 +65,10 @@ echo "Getting pub..."
 (set -x ; curl -fsSL https://github.com/devigned/pub/releases/download/v0.2.6/pub_v0.2.6_linux_amd64.tar.gz -o pub; tar -xzf pub)
 
 echo "Creating new SKU"
-(set -x ; ./pub_linux_amd64 skus put -p $PUBLISHER -o "$OFFER" -f sku.json ; echo "")
```
Comment (CecileRobertMichon) [inline images/capi/packer/azure/scripts/new-sku.sh:68]: Want to do the same thing for new disk version while you're here?

File: images/capi/packer/azure/scripts/new-sku.sh (hunk 1 above)
Comment (jsturtevant) [reply images/capi/packer/azure/scripts/new-sku.sh]: updated to echo -e "\nCreated sku"

**Review by CecileRobertMichon — APPROVED** [review]
/lgtm
/approve

**Pull request merged**